packages/core/src/__tests__/angular.test.ts

```diff
@@ -11,18 +11,3 @@ describe('Angular', () => {
     generator: componentToAngular,
   });
 });
-
-/*describe('Angular local', () => {
-  runTestsForTarget({
-    options: {
-      standalone: true,
-    },
-    target: 'angular',
-    generator: componentToAngular,
-    only: [
-      // 'basicRefAttributePassing',
-      'basicRefAttributePassingCustomRef'
-    ],
-    logOutput: true,
-  });
-});*/
```

Comment:
I see you use these for quick local testing. Will revert my removal 👍🏽 

```suggestion

/*describe('Angular local', () => {
  runTestsForTarget({
    options: {
      standalone: true,
    },
    target: 'angular',
    generator: componentToAngular,
    only: [
      // 'basicRefAttributePassing',
      'basicRefAttributePassingCustomRef'
    ],
    logOutput: true,
  });
});*/
```